types/src/transaction/script.rs

```diff
@@ -251,15 +251,16 @@ pub struct ScriptFunction {
     module: ModuleId,
     function: Identifier,
     ty_args: Vec<TypeTag>,
-    args: Vec<TransactionArgument>,
+    #[serde(with = "vec_bytes")]
+    args: Vec<Vec<u8>>,
```

Comment:
Quick question: what is special about `Vec<Vec<u8>>` that requires a custom serializer?

Reply:
To match the file format rule:

```
    - args:
        SEQ:
          SEQ: U8
```

We know that `Vec<u8>` needs to be serialized with `serde_bytes`, but `serde_bytes` does not support `Vec<Vec<u8>>`.